packages/rpc/src/__tests__/client.test.ts

```
@@ -0,0 +1,472 @@
+	});
+
+	describe("Error handling", () => {
+		it("should convert HTTP errors to HTTPException", async () => {
+			const mockResponse = {
+				ok: false,
+				status: 400,
+				text: mock(async () => "Bad Request"),
+			};
+
+			// Test the error handling logic
+			const jfetch = async (input: RequestInfo | URL, init?: RequestInit) => {
```

Biome (1.9.4)

[error] This parameter is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.
Unsafe fix: If this is intentional, prepend input with an underscore.


(lint/correctness/noUnusedVariables)